metaci/plan/models.py

```diff
@@ -45,14 +51,18 @@ class Plan(models.Model):
     yaml_config = models.TextField(null=True, blank=True, validators=[validate_yaml_field])
 
     class Meta:
-        ordering = ['name','repo__owner','repo__name', 'active', 'context']
+        ordering = ['name', 'active', 'context']
 
     def get_absolute_url(self):
         return reverse('plan_detail', kwargs={'plan_id': self.id})
 
     def __unicode__(self):
         return self.name
 
+    def get_repos(self):
```

Comment:
What's the reason for having this vs using the Django standard way of self.repos.all()?  I'm pretty sure self.repos.all() is also a generator.

Reply:
Actually, nevermind... it's to return the repository objects themselves